configs/embedder_nvlink.yaml

```
@@ -0,0 +1,69 @@
+  # Model configuration
+  model_name: jinaai/jina-embeddings-v4
+  device: cuda
+  use_fp16: true
+  
+  # Multi-GPU configuration for NVLink
+  multi_gpu: true                    # Enable multi-GPU processing
+  gpu_strategy: "dataparallel"       # Use PyTorch DataParallel
+  primary_device: "cuda:0"           # Primary GPU for coordination
+  device_ids: [0, 1]                 # Both RTX A6000s
+  enable_nvlink: true                # Explicitly use NVLink for GPU-GPU communication
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)